jobs/JJOB_DA_PREP

```diff
@@ -6,15 +6,15 @@
 # Prep SOCA-DA
 
 module purge
-module use -a /contrib/da/modulefiles
-module load jedi/jedim
+module use -a /scratch2/NCEPDEV/marine/marineda/modulefiles/
```

Comment:
@guillaumevernieres Did you save it here because we don't have writing privileges at /contrib/da/modulefiles ?

Reply:
@flampouris I assumed I didn't have the rights ... I should hve checked!